src/js/MainController.js

```diff
@@ -91,6 +93,7 @@ goog.require('ga_topic_service');
     // The main controller creates the OpenLayers map object. The map object
     // is central, as most directives/components need a reference to it.
     $scope.map = createMap();
+    gaMapload.init($scope);
```

Comment:
I think putting here the condition `if (gaPermalink.getParams().debug != 'true') ` makes more sense for future services.